test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -1507,6 +1516,33 @@ private void checkArgumentSequence(String[] expectedArguments,
 		}
 	}
 
+	@SuppressWarnings("boxing")
+	private void checkJarArgs(final String pathString) {
+		ProcessBuilder pb = makeProcessBuilder(null, new String[] {DASH_D_CMDLINE_ARG, "-jar", pathString}, null);
+		Map<String, String> env = pb.environment();
+		String javaToolOptionsArg = "-DjavaToolOptionsArg";
+		env.put(JAVA_TOOL_OPTIONS, javaToolOptionsArg );
+		String ibmJavaOptionsArg = "-DibmJavaOptionsArg";
+		env.put(IBM_JAVA_OPTIONS, ibmJavaOptionsArg );
+		ArrayList<String> actualArguments = runAndGetArgumentList(pb);
+		String fooBar = "-Dfoo=bar";
+		String longProp = "-Da.long.system.property=this is a long system property value to demonstrate long JVM arguments in the manifest file";
+		HashMap<String, Integer> posns = checkArguments(actualArguments, new String[] {javaToolOptionsArg, ibmJavaOptionsArg, DASH_D_CMDLINE_ARG, fooBar,
```

Comment:
When breaking a line like this, break it at a logical point:
```
HashMap<String, Integer> posns = checkArguments(actualArguments,
                new String[] {javaToolOptionsArg, ibmJavaOptionsArg, DASH_D_CMDLINE_ARG, fooBar, longProp});
```

Reply:
Okay.